upstream-community-operators/falco/falco-operator.v0.5.6.clusterserviceversion.yaml

```diff
@@ -0,0 +1,202 @@
+    mediatype: image/svg+xml
+  install:
```

Comment:
This operator probably needs RBAC, which should be defined in the CSV: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/building-your-csv.md#operator-install
This example above shows `permissions`, but if the operator needs any clusterPermissions, use `clusterPermissions` for those roles.